src/languageservice/download.ts

```diff
@@ -199,6 +199,8 @@ export default class ServiceDownloadProvider {
                 .then(inStream => {
                     this.install(inStream, installDirectory).then ( installed => {
                         resolve(installed);
+                    }).catch(installError => {
+                        reject(installError);
```

Comment:
the telemetry event for this will be sent in serviceClient 

---
In reply to: [91185859](https://github.com/Microsoft/vscode-mssql/pull/462#discussion_r91185859) [](ancestors = 91185859)